packages/ilib-loctool-json/test/JsonFile.test.js

```diff
@@ -2391,6 +2419,34 @@ describe("jsonfile", function () {
         expect(content).toBe(expected);
     });
 
+    test("Nested empty values should be preserved in default method", function () {
+        var jf = new JsonFile({
+            project: p,
+            pathName: "./json/nested-empty-items.json",
+            type: t
+        });
+        expect(jf).toBeTruthy();
+
+        // should read the file
+        jf.extract();
+
+        // no translations needed for this test
+        var translations = new TranslationSet();
+
+        var actual = jf.localizeText(translations, ["xx-YY"]);
```

Comment:
This tests use the "strings-schema" which doesn't attempt to localize any values in those properties. Is that the intention? Or should they be localizable and we're just testing that they are still okay when there is no value to localize?